examples/stable/test/end_to_end/candid_rpc/motoko_examples/superheroes/src/index.ts

```diff
@@ -13,7 +13,7 @@ const Superhero = IDL.Record({
     name: IDL.Text,
     superpowers: IDL.Opt(List)
 });
-type Superhero = {
+export type Superhero = {
```

Comment:
Use these types in the test code, do not use the automatically  generated candid types